src/audio/google_rtc_audio_processing_mock.c

```diff
@@ -13,6 +13,8 @@
 #include "ipc/topology.h"
 
 #define GOOGLE_RTC_AUDIO_PROCESSING_SAMPLE_RATE_HZ 48000
```

Comment:
this is now a config

Reply:
Well spotted. I now changed to using that.